.gitignore

```diff
@@ -60,6 +63,18 @@
 # build
 *build
 
+# images
+*.png
+*/*.png
+*/*/*.png
+*/*/*/*.png
+*/*/*/*/*.png
+*.pdf
+*/*.pdf
+*/*/*.pdf
+*/*/*/*.pdf
+*/*/*/*/*.pdf
+
 # pycache
 *pycache*
 */*pycache*
```

Comment:
Similarly these lines can be reduced to just:
```.gitignore
__pycache__/
```
which will match only directories named `__pycache__`, and will do so in any folder